<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispyVehicleTable</class>
 <widget class="QMainWindow" name="DispyVehicleTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1645</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Display vehicle Table</string>
  </property>
  <property name="toolTip">
   <string>DispyVehicleTable</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 170, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Looking for vehicle</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEditLookingForVehicle">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="placeholderText">
          <string>Please enter vehicle name or Manufacturer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_SearchForVehicle">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Search for vehicle</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Show vehicle status</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton_AllVehicles">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>all vehicles</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_AvailableVehicles">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>available vehicles</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButton_UnavailableVehicles">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>unavailable vehicles</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_ShowVehicles">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Show vehicles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidgetShowVehicles">
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>10</number>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>vehicle name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>manufactureer</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>construction year</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>km stood</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>condition</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>pieces</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>price</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>currency</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>availability</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1645</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
